frontend/tests/controllers/BadgesTest.php

```diff
@@ -13,7 +17,34 @@ class BadgesTest extends OmegaupTestCase {
     const QUERY_FILE = 'query.sql';
     const TEST_FILE = 'test.json';
 
+    public static function RunRequests($apicall) {
+        $identity = new stdClass();
```

Comment:
ya tenemos una clase que representa esto:

```suggestion
        $identity = new Identities([
            'username' => $apicall['username'],
            'password' => $apicall['password'],
        ]);
```

de paso, puedes cambiar `OmegaupTestCase::login` para que sólo acepte objetos de tipo `Identities`?

Y ahora como es súper obvio que eso es una identidad, puedes darle inline a L24:

```php
        $login = self::login(new Identities([
            'username' => $apicall['username'],
            'password' => $apicall['password'],
        ]));
```